recipe/meta.yaml

```
@@ -1,39 +1,39 @@
 {% set name = "ydata-profiling" %}
-{% set version = "4.8.3" %}
+{% set version = "4.16.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ydata-profiling-{{ version }}.tar.gz
-  sha256: d9467ecc8474344347ad89a6839db14762f0230e53df94f83a04810b2fa6cbd7
+  sha256: 4aa0d6412a140194396e34f5ffd348223f0319d877b0243e765f30ae633ed51e
 
 build:
+  number: 0
   entry_points:
     - ydata_profiling = ydata_profiling.controller.console:main
     - pandas_profiling = ydata_profiling.controller.console:main
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
-  number: 0
-  # numba is not available on s390x
-  skip: True   #  [py<37 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

```suggestion
  script_env:
    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```
Version is set to 0.0.dev0 in the logs.